ingestclient/client.py

```diff
@@ -164,6 +164,9 @@ def main(configuration=None, parser_args=None):
         check_version()
         return
 
+    # TODO Remove this line before committing to production
```

Comment:
Should you remove the line?

Reply:
👍

Reply:
Nice catch.  I actually updated this change but didn't check it in for some reason.